src/Http/Middleware/CsrfProtectionMiddleware.php

```diff
@@ -277,6 +281,10 @@ public function saltToken(string $token): string
             return $token;
         }
         $decoded = base64_decode($token, true);
+        if ($decoded === false) {
+            throw new RuntimeException('Invalid token data.');
```

Comment:
Should this be an InvalidArgumentException? It makes sense as RuntimeException in process(). Otherwise this looks good.

Reply:
Good idea.